frontend/src/pages/member/LoginPage.vue

```diff
@@ -73,12 +73,49 @@ export default {
         return;
       }
       try {
-        // TODO login API를 작성해주세요.
-        // const { email, password } = this.member;
-        // const data = await fetch("/login")
-        // TODO member 데이터를 불러와 주세요.
-        // const member = wait fetch("/members/me")
-        // this.setMember(member);
+        const {email, password} = this.member;
+        const token = await fetch("http://localhost:8080/login", {
```

Comment:
`http://localhost:8080` 이 부분은 실행 환경에 따라 달라질 수 있기 때문에 보통 Path만 이용해서 호출해요!
```suggestion
        const token = await fetch("/login", {
```

Reply:
해당 부분을 /api/login 같이 호출할 수 있도록 변경했어요!